### ISSUE TITLE
Configurable action types

### ISSUE THREAD

**Issue by gaearon** [omnidan/redux-undo#3]
I think it would've been nice if the Undo/Redo action types were configurable:

```js
combineReducers({
  counter: undoable(counter, {
    limit: 10,
    undoType: 'UNDO_COUNTER',
    redoType: 'REDO_COUNTER'
  })
})
```

This would let user have independently undoable parts of the tree.
(You can still provide default types and action creators for convenience.)

**Comment by omnidan** [comment]
Thanks for the suggestion, this is implemented in 0.2.5 :)

**Comment by Andsbf** [comment]
Hi, 

I know it is thread, but I didn't want to create a new one just to ask something.

is it possible to change the limit after the reducer has been  initialised? 

_Use case:_
My limit will be defined by the user's account type, so I need to set/change the reducer history limit after I have fetch the user's data.

Thanks in advance!
